# HG changeset patch
# User John W. Eaton <jwe@octave.org>
# Date 1523419090 14400
#      Tue Apr 10 23:58:10 2018 -0400
# Branch stable
# Node ID d12271fe1504ce5d084d0c52d037918e2031582b
# Parent  c80323fe49389dd54d4948923bedbf257a66c9d1
fix cut/paste error in previous change (bug #53599)

* symrec.h (symbol_record::symbol_record_rep::unbind_global_rep):
Forward to unbind_global_rep, not unbind_fwd_rep.

diff --git a/libinterp/corefcn/symrec.h b/libinterp/corefcn/symrec.h
--- a/libinterp/corefcn/symrec.h
+++ b/libinterp/corefcn/symrec.h
@@ -523,17 +523,17 @@ namespace octave
               {
                 // The rep this object points to is in the global
                 // scope, so delete the link to it.
 
                 m_fwd_scope = std::weak_ptr<symbol_scope_rep> ();
                 m_fwd_rep.reset ();
               }
             else
-              t_fwd_rep->unbind_fwd_rep ();
+              t_fwd_rep->unbind_global_rep ();
           }
       }
 
       std::shared_ptr<symbol_record_rep>
       dup (const std::shared_ptr<symbol_scope_rep>& new_scope) const;
 
       octave_value dump (context_id context) const;
 
